### ISSUE TITLE
Converting a playlist to Spotify

### ISSUE THREAD

**Issue by theg3ntlem4n** [matteobart/Twister#1]
Currently, I am working on an app that converts an array of Spotify song URIs into a playlist. I was wondering how you were able to import a playlist into Spotify? Could you show me how you authenticated the user as well as imported the playlist?

**Comment by matteobart** [comment]
Yeah I am currently working on this part now Spotify songs URIs into a playlist. The SpotifyKit pod that I am using does not support adding to playlists, so I'm working on that now. I have been a bit too busy for this project, but I seem to be getting back to it now. 
The user authentication was already done via SpotifyKit, but I made some slight adjustments. 
Cheers
